src/Sulu/Bundle/TestBundle/Testing/SuluTestCase.php

```diff
@@ -130,16 +130,21 @@ protected function createWebsiteClient()
     protected function initPhpcr()
     {
         /** @var SessionInterface $session */
-        $session = $this->getContainer()->get('doctrine_phpcr')->getConnection();
+        $registry = $this->getContainer()->get('doctrine_phpcr');
 
-        if ($session->nodeExists('/cmf')) {
-            NodeHelper::purgeWorkspace($session);
+        foreach ($registry->getConnections() as $session) {
+            if ($session->nodeExists('/cmf')) {
+                $session->getNode('/cmf')->remove();
+            }
             $session->save();
         }
 
+
         if (!$this->importer) {
-            $this->importer = new PHPCRImporter($session);
+            $this->importer = new PHPCRImporter($this->getContainer()->get('doctrine_phpcr')->getConnection());
         }
+        $this->getContainer()->get('sulu_document_manager.initializer')->initialize();
+        return;
```

Comment:
Not a fan of the below code which loads a raw PHPCR XML file. It is high maintenance and suseptable to error - e.g. if somebody updates and includes some "non initial" state in the content repository.

It also prevents attempts to test configurations with different "initial states" and is not bound to the actual configuration of the test suite.

Is the speed gained by such a thing worth the extra maintainence and risk of inconsistency? /cc @wachterjohannes @danrot @chirimoya